app/models.py

```diff
@@ -1,43 +1,51 @@
 import pytz
 import uuid
-
 from django.db import models
 from cloudinary.models import CloudinaryField
 
 
+def timezone():
+    # Pytz Timezone package http://pytz.sourceforge.net/
+    # https://stackoverflow.com/questions/47477109/how-to-store-timezones-efficiently-in-django-model
+    return sorted((item, item) for item in pytz.all_timezones)
+
+
+class Pronoun(models.Model):
+    PRONOUNS = [
+        ('They', 'They/Them'),
+        ('She', 'She/Her'),
+        ('He', 'He/Him')
+    ]
+    pronoun = models.CharField(
+        max_length=20,
+        choices=PRONOUNS
+    )
+
+
 class User(models.Model):
-    mentor = models.BooleanField(default=False)
-    mentee = models.BooleanField(default=True)
     user_id = models.UUIDField(
         primary_key=True,
         default=uuid.uuid4,
         editable=False
         )
+    role = models.ManyToManyField('Role', related_name='roles')
     display_name = models.CharField(
         blank=False, max_length=300,
-        help_text='Include first name and last name here')
-    about = models.TextField(max_length=1000, help_text='a brief description of you')
-    avatar = CloudinaryField('image')
-    stacks = models.ManyToManyField('Stack')
-
-    PRONOUN_CHOICES = [
-        ('They', 'They/Them'),
-        ('She', 'She/Her'),
-        ('He', 'He/Him')
-    ]
-    preferred_pronouns = models.CharField(
-        max_length=5,
-        choices=PRONOUN_CHOICES,
-        null=False
+        help_text='Include first name and last name here'
+    )
+    about = models.TextField(
+        max_length=1000,
+        help_text='a brief description of you'
     )
-    spoken_languages = models.ManyToManyField('SpokenLanguage')
+    avatar = CloudinaryField('image')
```

Comment:
Does the cloudinary field work with the current API setup? Or does this need to be commented out also, since it was commented out in the settings?

Reply:
well it has not thrown any errors so far, however through postman, I can't directly upload an image so I just append .png or .jpg file to a name when I'm testing. I'll uncomment it out in the settings, didn't know it was commented out

Reply:
yea of course what it needs is the name from the API. pls can you check whether you can upload an image that way? If you can't, please just make a new ticket for it so we don't hold down this PR longer

Reply:
right then, will do

Reply:
https://stackoverflow.com/questions/36805137/how-to-pass-options-to-cloudinaryfield-in-django-model